### PR TITLE
fix rtcon and tests

### PR DESCRIPTION
**Pull request by yosuke**
治せそうなところを治しました。

### CODE REVIEW

File: rtshell/rtcon.py [hunk 1]
```diff
@@ -134,6 +134,12 @@ def property_callback(option, opt, option_value, parser):
         print('{0}: No ports specified.'.format(os.path.basename(sys.argv[0])),
                 file=sys.stderr)
         return 1
+
+    if len(args) < 2:
```
Comment (gbiggs) [inline rtshell/rtcon.py]: A connection is allowed to have a single port. rtcon must allow this use case. For example, the DDS port for RTM relies on using rtcon to connect a single port to a topic.

File: test/test_cmds.py [hunk 2]
```diff
@@ -1155,6 +1162,8 @@ def _check_rtsys_xml(self, rtsys):
         rtsprofile.rts_profile.RtsProfile(xml_spec=rtsys)
 
     def _check_rtsys_yaml(self, rtsys):
+        print 'check rtsys yaml:'
```
Comment (gbiggs) [inline test/test_cmds.py]: Should this and the next line be deleted?

Comment (yosuke) [inline test/test_cmds.py]: I want to keep this because it works nicely with travis-ci.

File: rtshell/rtcon.py (hunk 1 above)
Comment (yosuke) [inline rtshell/rtcon.py]: Fixed(reverted) in this commit.

https://github.com/yosuke/rtshell/commit/adee1aea216e3772db87351d03380bea77147060

**Pull request merged**